tokio-tcp/src/listener.rs

```diff
@@ -1,21 +1,20 @@
-use net::tcp::Incoming;
-use net::tcp::TcpStream;
+use super::Incoming;
+use super::TcpStream;
 
 use std::fmt;
 use std::io;
 use std::net::{self, SocketAddr};
 
 use futures::{Poll, Async};
 use mio;
-
-use reactor::{Handle, PollEvented2};
+use tokio_reactor::{Handle, PollEvented};
 
 /// An I/O object representing a TCP socket listening for incoming connections.
 ///
 /// This object can be converted into a stream of incoming connections for
 /// various forms of processing.
 pub struct TcpListener {
-    io: PollEvented2<mio::net::TcpListener>,
+    io: PollEvented<mio::net::TcpListener>,
```

Comment:
That's not only splitting tokio::net module into 2 additional crates

Reply:
What do you mean by that? The `tokio` crate re-exports `tokio_reactor::PollEvented` as `PollEvented2`, so this is exactly the same type.

Reply:
OK